Makefile

```diff
@@ -99,6 +99,8 @@ check-changelog: $(BEST_PY3)
 	$(BEST_PY3) scripts/check-changelog.py
 
 deploy: $(TOOL_VIRTUALENV)
+	openssl aes-256-cbc -K $encrypted_39cb4cc39a80_key -iv $encrypted_39cb4cc39a80_iv -in secrets.tar.enc -out secrets.tar -d
```

Comment:
To double-check: I know environment variables in .travis.yml and Makefiles are both weird – are you sure this works like you expect?